src/containers/Projects/ProjectRulesAndGuide/MilestonesAndPayouts/MilestonesAndPayouts.scss

```diff
@@ -0,0 +1,222 @@
+        margin-right: 12px;
+      }
+    }
+
+    &-text {
+      @extend %h4;
+      color: var(--color-gray-700);
+      margin: 0;
+    }
+  }
+
+  &__pow {
+    margin-bottom: 88px;
+
+    &-heading {
+      @extend %h2;
+      color: var(--color-gray-900);
+      font-weight: var(--font-weight-bold);
+      margin-bottom: 64px;
+    }
+
+    &-table {
+      &-row {
+        box-shadow: inset 0px -1px 1px #e6e6e6;
+        display: grid;
+        gap: 32px;
+        grid-template-columns: 160px auto 160px;
+        padding: 32px 40px;
+
+        &-apps {
+          align-self: center;
+          display: flex;
+          justify-self: end;
+
+          &-logo {
+            align-items: center;
+            background: var(--color-blue-50);
+            border-radius: 50%;
+            display: flex;
+            height: 56px;
+            justify-content: center;
+            width: 56px;
+          }
+        }
+
+        &-heading {
+          @extend %h3;
+          align-self: center;
+          border-right: 1px solid var(--color-gray-075);
+          color: var(--color-blue-900);
+          padding-right: 8px;
+        }
+
+        &-text {
+          @extend %h3;
+          align-self: center;
+          color: var(--color-gray-700);
+          margin: 0;
+        }
+      }
+    }
+  }
+
+  &__process {
+    margin-bottom: 120px;
+
+    &-heading {
+      @extend %h2;
+      color: var(--color-gray-900);
+      font-weight: var(--font-weight-bold);
+      margin-bottom: 64px;
+    }
+
+    &-step {
+      box-shadow: inset 0px -1px 1px #e6e6e6;
+      display: grid;
+      gap: 32px;
+      grid-template-columns: 56px auto;
+      padding: 32px;
+
+      &-text {
+        @extend %h3;
+        align-self: center;
+        color: var(--color-gray-700);
+      }
+    }
+  }
+
+  &__roadmap {
+    box-shadow: 0px 0px 30px rgba(229, 234, 244, 0.48);
+    margin-bottom: 120px;
+    padding: 40px;
+
+    &-heading {
+      @extend %h2;
+      color: var(--color-gray-900);
+      font-weight: var(--font-weight-bold);
+      margin-bottom: 8px;
+    }
+
+    &-illustration {
+      height: auto;
+      width: 100%;
+    }
+
+    &-text {
+      @extend %h3;
+      color: var(--color-gray-700);
+      margin-bottom: 88px;
+    }
+  }
+
+  @media (max-width: 992px) {
+    &__roadmap {
+      padding: 48px;
+    }
+  }
+
+  @media (max-width: 768px) {
+    &__pow {
+      &-table {
+        &-row {
+          padding: 28px 32px;
+        }
+      }
+    }
+
+    &__process {
+      &-divider {
+        border-top: 1px solid #e6e6e6;
+        margin: 0;
+        width: 60%;
+      }
+
+      &-step {
+        box-shadow: none;
+        padding: 32px 0;
+      }
+    }
+
+    &__roadmap {
+      padding: 32px;
+    }
+  }
+
+  @media (max-width: 480px) {
+    &__heading {
+      @include h1;
+    }
+
+    &__pow {
+      &-table {
+        &-row {
+          box-shadow: none;
+          grid-template-areas: 'heading heading' 'text apps';
```

Comment:
wait what is this.... never seen this before hahaha

Reply:
here:
[https://www.w3schools.com/CSSref/pr_grid-template-areas.asp](
https://www.w3schools.com/CSSref/pr_grid-template-areas.asp)